doc/source/serve/http-servehandle.rst

```diff
@@ -215,11 +215,30 @@ With :ref:`model wrappers<air-serve-integration>`, you can specify it via the ``
         http_adapter=json_to_ndarray
     )
 
+.. note::
+
+    Serve also support pydantic model as short-hand for http adapter in model wrapper. Instead of function,
+    you can directly pass in a pydantic model class to mean "validate http body with this schema".
+    Once validated, the model instances will passed to predictor.
```

Comment:
```suggestion
    Serve also supports pydantic models as a short-hand for HTTP adapters in model wrappers. Instead of functions,
    you can directly pass in a pydantic model class to mean "validate the HTTP body with this schema".
    Once validated, the model instance will passed to the predictor.
```
please try to be more careful about grammar

Reply:
😅 tyty